client/src/components/modals/nr-not-required.vue

```diff
@@ -1,29 +1,35 @@
 <template>
   <!-- NB: dummy modal attach point so unit tests can see it -->
-  <v-dialog v-model="showModal" max-width="40%" attach="">
-    <v-card class="pa-9">
-      <v-card-text class="h3">You don't need a Name Request if...</v-card-text>
+  <v-dialog v-model="showModal" width="50rem" attach="">
+    <v-card class="pa-9 nr-not-required-dialog">
```

Comment:
Invalid class. This modal should look like the Receipts modal, Cancel modal, etc. (40px gutter all around)